crates/ruff_python_parser/tests/snapshots/invalid_syntax@expressions__named__missing_expression_2.py.snap

```diff
@@ -0,0 +1,122 @@
+---
+source: crates/ruff_python_parser/tests/fixtures.rs
+input_file: crates/ruff_python_parser/resources/invalid/expressions/named/missing_expression_2.py
+---
+## AST
+
+```
+Module(
+    ModModule {
+        range: 0..87,
+        body: [
+            Expr(
+                StmtExpr {
+                    range: 61..71,
+                    value: Named(
+                        ExprNamed {
+                            range: 62..71,
+                            target: Name(
+                                ExprName {
+                                    range: 62..63,
+                                    id: "x",
+                                    ctx: Store,
+                                },
+                            ),
+                            value: Name(
+                                ExprName {
+                                    range: 68..71,
+                                    id: "def",
+                                    ctx: Load,
+                                },
+                            ),
+                        },
+                    ),
+                },
+            ),
+            AnnAssign(
+                StmtAnnAssign {
+                    range: 72..87,
+                    target: Call(
+                        ExprCall {
+                            range: 72..77,
+                            func: Name(
+                                ExprName {
+                                    range: 72..75,
+                                    id: "foo",
+                                    ctx: Load,
+                                },
+                            ),
+                            arguments: Arguments {
+                                range: 75..77,
+                                args: [],
+                                keywords: [],
+                            },
+                        },
+                    ),
+                    annotation: Name(
+                        ExprName {
+                            range: 83..87,
+                            id: "pass",
+                            ctx: Load,
+                        },
+                    ),
+                    value: None,
+                    simple: false,
+                },
+            ),
+        ],
+    },
+)
+```
+## Errors
+
+  |
+3 | (x :=
+4 | 
+5 | def foo():
+  | ^^^ Syntax Error: Expected an identifier, but found a keyword 'def' that cannot be used here
```

Comment:
What's the reason that it expected an `identifier`? Is it because this is the default case of expression parsing? It might be fine but it's slightly misleading because it gives the impression that only an identifier (and not any arbitrary expression) is valid.

We don't need to address this as part of this PR (or even as part of this phase)

Reply:
I agree that it's misleading. I was confused myself. You're correct that this is the default case if we don't find any valid start token for an expression. I'll add it to error recovery issue.